2024/03/13 22:44:10 [error] 19660#10420: *7 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://[::1]:1001/", host: "localhost"
2024/03/13 22:44:12 [error] 19660#10420: *7 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:1001/", host: "localhost"
2024/03/13 22:44:14 [error] 19660#10420: *7 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://[::1]:1002/", host: "localhost"
2024/03/13 22:44:16 [error] 19660#10420: *7 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:1002/", host: "localhost"
2024/03/13 22:44:16 [error] 19660#10420: *7 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://webdevtask/favicon.ico", host: "localhost", referrer: "http://localhost/"
2024/03/13 22:44:28 [error] 19660#10420: *7 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://[::1]:1002/", host: "localhost"
2024/03/13 22:44:30 [error] 19660#10420: *7 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:1002/", host: "localhost"
2024/03/13 22:44:32 [error] 19660#10420: *7 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:1001/", host: "localhost"
2024/03/13 22:44:34 [error] 19660#10420: *7 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://[::1]:1001/", host: "localhost"
2024/03/13 22:44:34 [error] 19660#10420: *7 no live upstreams while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://webdevtask/favicon.ico", host: "localhost", referrer: "http://localhost/"
2024/03/13 22:45:11 [error] 19660#10420: *7 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://[::1]:1002/", host: "localhost"
2024/03/13 22:45:13 [error] 19660#10420: *7 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:1002/", host: "localhost"
2024/03/13 22:45:15 [error] 19660#10420: *7 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://[::1]:1001/", host: "localhost"
2024/03/13 22:47:31 [error] 19660#10420: *25 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://[::1]:1002/", host: "localhost"
2024/03/13 22:47:40 [error] 19660#10420: *26 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /ho,e HTTP/1.1", upstream: "http://127.0.0.1:1002/ho,e", host: "localhost"
2024/03/13 22:47:46 [error] 19660#10420: *26 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /home HTTP/1.1", upstream: "http://[::1]:1001/home", host: "localhost"
2024/03/13 22:48:50 [error] 19660#10420: *41 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /home HTTP/1.1", upstream: "http://[::1]:1001/home", host: "localhost"
2024/03/13 22:48:53 [error] 19660#10420: *41 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /home HTTP/1.1", upstream: "http://[::1]:1002/home", host: "localhost"
2024/03/13 23:17:37 [emerg] 14892#19220: cannot load certificate "D:/path/to/your/ssl_certificate.crt": BIO_new_file() failed (SSL: error:02001003:system library:fopen:No such process:fopen('D:/path/to/your/ssl_certificate.crt','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2024/03/13 23:19:35 [emerg] 16056#15584: cannot load certificate "D:\Programming\Python\nthPrimeNumberFinder\nginx-1.24.0/conf/certificate.crt": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('D:\Programming\Python\nthPrimeNumberFinder\nginx-1.24.0/conf/certificate.crt','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2024/03/13 23:19:56 [emerg] 8004#10820: cannot load certificate "D:\Programming\Python\nthPrimeNumberFinder\nginx-1.24.0/conf/certificate.crt": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('D:\Programming\Python\nthPrimeNumberFinder\nginx-1.24.0/conf/certificate.crt','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2024/03/13 23:20:06 [emerg] 16924#19216: cannot load certificate "D:\Programming\Python\nthPrimeNumberFinder\nginx-1.24.0/conf/certificate.crt": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('D:\Programming\Python\nthPrimeNumberFinder\nginx-1.24.0/conf/certificate.crt','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2024/03/13 23:20:48 [emerg] 14436#11820: cannot load certificate "D:\Programming\Python\nthPrimeNumberFinder\nginx-1.24.0/conf/certificate.crt": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('D:\Programming\Python\nthPrimeNumberFinder\nginx-1.24.0/conf/certificate.crt','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2024/03/13 23:20:48 [emerg] 16876#3916: cannot load certificate "D:\Programming\Python\nthPrimeNumberFinder\nginx-1.24.0/conf/certificate.crt": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('D:\Programming\Python\nthPrimeNumberFinder\nginx-1.24.0/conf/certificate.crt','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2024/03/13 23:20:48 [emerg] 5700#3324: cannot load certificate "D:\Programming\Python\nthPrimeNumberFinder\nginx-1.24.0/conf/certificate.crt": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('D:\Programming\Python\nthPrimeNumberFinder\nginx-1.24.0/conf/certificate.crt','r') error:2006D080:BIO routines:BIO_new_file:no such file)
2024/03/13 23:21:33 [error] 9332#3876: *2 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /home HTTP/1.1", upstream: "http://[::1]:1002/home", host: "localhost"
